src/visitors/visitor_utils.cpp

```diff
@@ -35,6 +35,11 @@ std::string get_new_name(const std::string& name,
     return (name + "_" + suffix + "_" + std::to_string(counter));
 }
 
+std::shared_ptr<ast::Name> get_name_from_string(const std::string& name) {
```

Comment:
True. It's not necessary. I had added it because this was needed in a commit but then removed the related code. We can remove it to make it clearer but maybe it could be used at some point in the future. As you wish

Reply:
If it is not used, then it is dead code. At least, it should be called/validated in the tests.

Reply:
I didn't think of it this way Tristan. You're right. This should go away 👍